packages/kbn-monaco/src/esql/lib/ast/autocomplete/complete_items.ts

```diff
@@ -100,7 +103,7 @@ function buildCharCompleteItem(
   return {
     label,
     insertText: quoted ? `"${label}"` : label,
-    kind: 1,
+    kind: 11,
```

Comment:
nit: I had to look up the type and then the enum to understand that this changes the kind from `Function` to `Operator`. That's okay, but is there any reason we don't just use the enum directly?

Reply:
Same reason as above, bundle size.